src/assets/extra.json

```diff
@@ -0,0 +1,72 @@
+[
```

Comment:
hmm, don't think we want the manifest to live here. I think it should still be pulled from release and looked up somehow